.github/SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security policy
+ 
+## Scope
+
+Please only report an issue here if it relates specifically to the Plutus Core language or compiler.
+This is also an appropriate place to report issues that stem from the EUTXO ledger model.
+
+Note that many security issues will instead appear in specific applications or in the [libraries that support them](https://github.com/input-output-hk/plutus-apps). 
+
+## Weaknesses
+
+If you have found a _weakness_ (in the sense of the [CWE](https://cwe.mitre.org/); a source of potential vulnerabilities) that affects users of this library, please open a "Security report" issue and describe it as a weakness.
```

Comment:
Now the only path to raise a security report is for a weaknesses and not vulnerabilities, so should we change form's labels to add/change to "weakness"?

Reply:
I think there might also be stuff that doesn't clearly fall into either category, so I'm happy to leave it a bit generic for now.